Sources/Application/extensions/Guess+Equatable.swift

```diff
@@ -0,0 +1,6 @@
+extension Guess: Equatable {
+}
+
+public func ==(l: Guess, r: Guess) -> Bool {
+	return l.currentGuess == r.currentGuess
+}
```

Comment:
I was thinking, if there's no reason for the Guess to be equatable in the game, you could move this protocol extension into the Application's test package.